acmwebsite/controllers/schedule.py

```diff
@@ -0,0 +1,42 @@
+import datetime
+
+from icalendar import Calendar, Event
+from tg import expose, response
+
+from acmwebsite.lib.base import BaseController
+from acmwebsite.lib.helpers import log
+from acmwebsite.model import DBSession, Meeting
+
+
+class ScheduleController(BaseController):
+    def __init__(self):
+        self.meetings = DBSession.query(Meeting).order_by(Meeting.date).all()
+
+    @expose('acmwebsite.templates.schedule')
+    @expose('json', exclude_names=['page'])
```

Comment:
`json` api can be removed now that we have iCal... nothing uses it now so let's get rid of it while it's still unused!